includes/admin/class-wcmypa-admin.php

```diff
@@ -113,6 +118,91 @@ public function __construct()
         add_action("woocommerce_process_product_meta", [$this, "productOptionsFieldSave"]);
     }
 
+    /**
+     * @throws \Exception
+     */
+    public function restrictManagePosts(): void
```

Comment:
Als je dit een duidelijkere naam geeft is die comment bij de hooks niet nodig.

Reply:
done